packages/grid/src/Grid.tsx

```diff
@@ -2050,8 +2057,11 @@ class Grid extends PureComponent<GridProps, GridState> {
 
     this.setViewState({ top, left, leftOffset, topOffset });
 
-    event.stopPropagation();
-    event.preventDefault();
+    // Check if we're at the top or bottom of the grid
+    if (top >= 0 && topOffset !== 0) {
```

Comment:
Scrolling seems to be broken in `IrisGrid` sample in styleguide. I think something isn't working as expected here. Also, a little confused how this is checking top or bottom? Is this a stale comment, or is there something missing in the code here?

Reply:
Taking a look rn